packages/icon/src/index.ts

```diff
@@ -2,6 +2,7 @@ import createIconComponent from './createIconComponent';
 import glyphs from './glyphs';
 
 export { default as createIconComponent } from './createIconComponent';
+export { default as createGlyphComponent } from './createGlyphComponent';
```

Comment:
Should we also export the `LGGlyph` namespace now, since that contains the interface for the components generated by the function?